javascript/extractor/src/com/semmle/js/extractor/trapcache/ITrapCache.java

```diff
@@ -18,4 +22,29 @@ public interface ITrapCache {
    *     cached information), or does not yet exist (and should be populated by the extractor)
    */
   public File lookup(String source, ExtractorConfig config, FileExtractor.FileType type);
+
+  /**
+  * Build a TRAP cache as defined by the extractor options, which are read from the corresponding
+  * environment variables as defined in
+  * https://github.com/github/codeql-core/blob/main/design/spec/codeql-extractors.md
+  *
+ * @return a TRAP cache
+ */
+  public static ITrapCache fromExtractorOptions() {
```

Comment:
```suggestion
  *
  * @return a TRAP cache
  */
  public static ITrapCache fromExtractorOptions() {
```
Not sure what happened to the indentation there

Reply:
Good spot, fixed.